tools/send-to-caas/send-utils.js

```diff
@@ -173,13 +173,22 @@ const getDateProp = (dateStr, errorMsg) => {
   }
 };
 
-const getOrigin = () => {
-  const origin = getConfig().project || getConfig().repo;
+const processRepoForFloodgate = (repo, fgColor) => {
+  if (repo && fgColor) {
+    return repo.slice(0, repo.lastIndexOf(`-${fgColor}`));
+  }
+  return repo;
+};
+
+const getOrigin = (fgColor) => {
+  const bulkPublisherRepo = processRepoForFloodgate(getConfig().repo, fgColor);
+  const origin = getConfig().project || bulkPublisherRepo;
```

Comment:
nit - since we only need `bulkPublisherRepo` when `project` isn't defined
```suggestion
  const { project, repo } = getConfig();
  const origin = project || processRepoForFloodgate(repo, fgColor);
```

Reply:
Thanks for checking @chrischrischris, updated this.